simulationworkflowschema/equation_of_state.py

```diff
@@ -158,8 +161,28 @@ def normalize(self, archive, logger):
             self.results = EquationOfStateResults()
             self.outputs.append(Link(name=WORKFLOW_RESULTS_NAME, section=self.results))
 
+        task0_archive = self.tasks[0].task.m_root()
+        tasks = self.tasks[:]
+        if task0_archive.workflow2:
+            # remove the first task if it is a GeometryOptimization (not part of the EOS)
+            if isinstance(task0_archive.workflow2, GeometryOptimization):
```

Comment:
should we instead include only the single point tasks?

Reply:
yeah I considered that, I wasn't sure since I don't really know about the context of these calculations. If the E-V curve should always be calculated from SP calcs, then yeah I would say that makes sense